src/test/java/org/jenkinsci/plugins/matrixauth/Security2180Test.java

```diff
@@ -43,9 +47,26 @@ public class Security2180Test {
     private static final Map<Permission, Set<String>> ANONYMOUS_CAN_ITEM_READ =
             Collections.singletonMap(Item.READ, Collections.singleton("anonymous"));
 
+    private static final Logger LOGGER = Logger.getLogger(Security2180Test.class.getName());
+
+    @ClassRule
+    public static BuildWatcher watcher = new BuildWatcher();
```

Comment:
What is this for? https://github.com/jenkinsci/jenkins-test-harness/blob/a7b41a28620768c3f3b1a6bbcb909108e03cbb21/src/main/java/org/jvnet/hudson/test/BuildWatcher.java#L43 makes it look more like a test development utility.

Reply:
Well yes, but you might some day wish to edit the test.